wails-frontend/src/const/networks.ts

```diff
@@ -0,0 +1,15 @@
+export const MAINNET = 'MainNet';
+export const BUILDNET = 'BuildNet';
+export const SECURENET = 'SecureNet';
+export const LABNET = 'LabNet';
+export const SANDBOX = 'Sandbox';
+
+export const CHAIN_ID_TO_NETWORK_NAME = {
+  77658377: MAINNET,
+  77658366: BUILDNET,
+  77658383: SECURENET,
+  77658376: LABNET,
+  77: SANDBOX,
+} as const; // type is inferred as the specific, unchangeable structure
```

Comment:
Shouldn't this be part of Massa Web3 or another TS library ? 
That would allow us to not have to copy paste this everywhere and have issues if we update it in a repository and not in another 

Note that this isn't blocking for this PR

Reply:
Which one of those libs does Massa station wallet currently depend on ?

Reply:
massa-web3 and react-ui-kit

Reply:
Then it makes sense to add it to massa-web3, probably in this PR: https://github.com/massalabs/massa-web3/pull/510 so that it provides this table to all depending projects.
What do you think ?